mdx2/command_line/import_data.py

```
@@ -2,51 +2,91 @@
+    slices = [sl for sl in image_series.chunk_slice_along_axis(0)]
+
+    # Access virtual dataset information through public API
+    # Reload the ImageSeries to ensure we have the updated virtual dataset
+    image_series_reloaded = ImageSeries.load(outfile)
+    files = image_series_reloaded.virtual_source_files
+    vpath = image_series_reloaded.virtual_dataset_path
+
+    # Properties will raise RuntimeError if internal API has changed
+    # If data is not a virtual dataset, they return None
+    if files is None or vpath is None:
+        raise RuntimeError(f"Expected virtual dataset in {outfile}, but virtual dataset information is not available.")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)